controllers/provisioners/eks/create.go

```diff
@@ -89,6 +89,14 @@ func (ctx *EksInstanceGroupContext) CreateScalingGroup(lcName string) error {
 	}
 	ctx.Log.Info("created scaling group", "instancegroup", instanceGroup.GetName(), "scalinggroup", asgName)
 
+	if len(spec.EKSConfiguration.SuspendedProcesses) > 0 {
```

Comment:
We already do a len check in the actual methods, is it possible to just skip this if and always call UpdateScalingProcesses?
Minor nit: maybe nicer to call it as:
```
if err := ctx. UpdateScalingProcesses(asgName); err != nil {
		return err
}
```
you can probably move the logging to inside `UpdateScalingProcesses` and also specify what changed.
e.g.
```
ctx.Log.Info("suspended scaling processes", "instancegroup", instanceGroup.GetName(), "scalinggroup", asgName, "difference", diff)
---
ctx.Log.Info("resumed scaling processes", "instancegroup", instanceGroup.GetName(), "scalinggroup", asgName, "difference", diff)
```

Reply:
Done 👍